docs/api.rst

```diff
@@ -10134,13 +10129,12 @@ Dealing with ignored assets
    :resjson list result: A list of asset names that are currently ignored.
    :statuscode 200: Assets successfully added
    :statuscode 400: Provided JSON or data is in some way malformed.
-   :statuscode 401: User is not logged in.
-   :statuscode 409: One of the assets provided is already on the list.
+   :statuscode 401: User is not logged in. One of the assets provided is already on the list.
```

Comment:
Hmm no again here we discussed this. 401 is **only** for user not bein glogged in. Other errors are other codes. In this case 409.

Reply:
I reverted the commit and didn't notice this ones. fixing it